packages/dicomImageLoader/src/decodeImageFrameWorker.js

```diff
@@ -331,8 +384,14 @@ function createDestinationImage(
   };
 }
 
-/** Scales the image frame, updating the frame in place with a new scaled
- * version of it (in place modification)
+/**
+ *
+ * @description Scales the image frame, updating the frame in place with a new
+ * scaled version of it (in place modification)
+ * @param {import("@cornerstonejs/core").Types.IImageFrame} imageFrame
```

Comment:
Can you rename the file to decodeImageFrameWorker.ts so you can directly declare at least some of the relevant types?

Reply:
done and updated